data/processed/hesa/total_postgraduates.yaml

```diff
@@ -0,0 +1,22 @@
+api: the type of API to be used to fetch data: Download
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/students/table-13#note
+description: Total number of postgraduate students in an NUTS2 region
+endpoint_source: https://www.hesa.ac.uk/data-and-analysis/students/table-13.csv
+framework_group: 1
+is_experimental: False
+nuts_level: 2
+source: the data provider: HESA (Higher Education Statistical Agency) https://www.hesa.ac.uk/
+year_range: ['2014/2015', '2018/19']
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: Total number of full-time students enrolled for postgraduate qualifications in universities in the area
+    id: total_postgraduates
+    label: Total number of postgraduate students
+    type: int
+  year:
+    data_type: str
+    label: Academic year
```

Comment:
`Start of the academic year` or `End of the academic year`? (we need to define or use a convention here)